spec.md

```diff
@@ -167,24 +167,38 @@ unsecured loan customers.
 
 Event (root element)
 
-| Name            | C.   | Type        | V. | Remark                          |
-|-----------------|------|-------------|----|---------------------------------|
-| application     | 1    | Application | v0 | Contains the loan application   |
-| broker          | 0..1 | reverse-dns | v0 | Domain-name of the broker       |
-| brokerReference | 0..1 | string      | v0 | Per-broker unique, not globally |
+| Name                  | C.   | Type                      | V. | Remark                                   |
+|-----------------------|------|---------------------------|----|------------------------------------------|
+| application           | 1    | Application               | v0 | Contains the loan application            |
+| broker                | 0..1 | reverse-dns               | v0 | Domain-name of the broker                |
+| brokerReference       | 0..1 | string                    | v0 | Per-broker unique, not globally          |
+| dataProtectionContext | 1    | DataProtectionContext | v0 | Sets data-protection rules for the event |
 
 Application
 
 | Name            | C.   | Type           | V. | Remark                              |
 |-----------------|------|----------------|----|-------------------------------------|
 | loanAmount      | 1    | number         | v0 | Total N/o SEK applied for           |
-| termYears       | 1    | number         | v0 | N/o desired years term for loan     |
+| termMonths      | 1    | number         | v0 | N/o desired years term for loan     |
 | refinanceAmount | 0..1 | number         | v0 | N/o SEK which are refinanced        |
 | applicant       | 1    | Applicant      | v0 | Main applicant                      |
 | coApplicant     | 0..1 | Applicant      | v0 | co-Applicant                        |
 | loanPurpose     | 1    | LoanPurpose    | v0 | Primary purpose of getting the loan |
 | extensions      | 0..1 | ExtensionPoint | v0 |                                     |
 
+DataProtectionContext
+
+| String value | Remark                                                   |
+|--------------|----------------------------------------------------------|
+| REAL         | Production data consisting concerning real data-subjects |
+| FICTIONAL    | Fictional data, does not concern real data-subjects      |
+
+Consumers MAY reject or refuse to process data sent in the `REAL` data
```

Comment:
While Consumer is technically correct i would preffer the wording receiver or recipient due to the ambiguous meaning of the word consumer in this context

Reply:
```
The standard defines the behaviour of two actors, the producer and the
consumer, as defined in the CloudEvents specification```
Rhee

Reply:
I agree on this point though.

Reply:
#22

Reply:
I know that it is defined in the standard, but sometime the standard is not perfectly applicable to every use-case.

Reply:
Agree but moving this to #22

Reply:
Given that this is done in #22 can we merge?